tests/mongodb/test_gene_functions.py

```diff
@@ -0,0 +1,121 @@
+"""The core Python code for generating data."""
+import pytest
+from src.data_import.data_import import get_dataset_path
+
+from src.data_import.mongodb.gene_functions import *
+from src.data_import.mongodb.mongo_wrapper import MongoWrapper
+
+GENE_LIST_DATASET = 'example_gene_list.txt'
+
+RNASeq_RECORD = {"_id": "5a7519801fd85c0e41c94c51",
+                 "gene": "ENSG00000232977",
```

Comment:
Do not indent like this, instead create a newline and use 4 spaces. See style here:

http://edgepy.readthedocs.io/en/latest/CONTRIBUTING.html#documentation-and-code-style

Reply:
Consider using a pretty JSON formatter in your IDE to give you a headstart.